generator/internal/genclient/translator/protobuf/protobuf.go

```diff
@@ -55,29 +55,30 @@ type Options struct {
 
 // Translate translates proto representation into a [genclienGenerateRequest].
 func Translate(req *pluginpb.CodeGeneratorRequest, opts *Options) (*genclient.GenerateRequest, error) {
-	api := makeAPI(req)
-
 	codec, err := language.NewCodec(opts.Language)
 	if err != nil {
 		return nil, err
 	}
+	var name string
+	if opts.ServiceConfig != nil {
+		name = opts.ServiceConfig.Name
```

Comment:
I think this is an improvement, but I am not sure it is exactly what we want.  For our test service that would be "secretmanager.googleapis.com".  Or maybe we want this, but we strip out the `.googleapis.com` to create crate names and the such.

No action required.

Reply:
Ack that's a good point.